src/language/walker/ruleWalker.ts

```diff
@@ -82,6 +82,14 @@ export class RuleWalker extends SyntaxWalker {
         }
     }
 
+    public createFix(description: string, replacements: Replacement[]): Fix {
+        return new Fix(this.ruleName, description, replacements);
+    }
+
+    public createReplacement(start: number, length: number, text: string): Replacement {
+        return new Replacement(start, length, text);
+    }
```

Comment:
in the future, we could possible add some helper methods for typical fixes (`insertAt`, `delete`, stuff like that).